qutebrowser/browser/commands.py

```diff
@@ -849,6 +850,11 @@ def tab_focus(self, index: {'type': (int, 'last')}=None, count=None):
         if index is None and count is None:
             self.tab_next()
             return
+        if index:
+            try:
+                index = int(index)
+            except ValueError:
+                raise cmdexc.CommandError("tab-focus takes an int or 'last' not: {}".format(index))
```

Comment:
What's the difference between checking this here and the way it was before (via the argument annotation, `index: {'type': (int, 'last')}=None`)?

(I didn't actually check, but they should do the exact same thing, no?)